packages/renderer/src/lib/network/CreateNetwork.svelte

```diff
@@ -0,0 +1,321 @@
+        <Input
+          bind:value={networkInfo.ipRange}
+          name="ipRange"
+          id="ipRange"
+          placeholder="IP range (e.g., 10.89.0.0/25 or 2001:db8::/64)"
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label for="gateway" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]">Gateway</label>
+        <Input
+          bind:value={networkInfo.gateway}
+          name="gateway"
+          id="gateway"
+          placeholder="Gateway (e.g., 10.89.0.1 or 2001:db8::1)"
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label for="internal" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Internal</label>
+        <SlideToggle id="internal" name="internal" bind:checked={networkInfo.internalEnabled} />
+      </div>
+
+        <div hidden={createNetworkInProgress}>
+        <label for="driver" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]">Driver</label>
+        <Dropdown
+          id="driver"
+          name="driver"
+          bind:value={networkInfo.driver}
+          options={availableDrivers}
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label
+          for="options"
+          class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Advanced Options (driver-specific):</label
+        >
+      
+        {#each networkInfo.options as _, index (index)}
+          <div class="flex flex-col w-full py-1">
+            <div class="flex flex-row justify-center items-center w-full">
+              <Input
+                bind:value={networkInfo.options[index]}
+                placeholder="e.g., com.docker.network.bridge.name=mybr0"
+                class="ml-2 w-full" />
+
+              <Button
+                type="link"
+                hidden={index === networkInfo.options.length - 1}
+                on:click={(): void => deleteOption(index)}
+                icon={faMinusCircle} />
+
+              <Button
+                type="link"
+                hidden={index < networkInfo.options.length - 1}
+                on:click={addOption}
+                icon={faPlusCircle} />
+            </div>
+          </div>
+        {/each}
+      </div>
+
+      <div class="w-full flex flex-row space-x-4">
+        <Button
+          disabled={hasInvalidFields || createNetworkInProgress || createNetworkFinished}
+          inProgress={createNetworkInProgress}
+          class="w-full"
+          hidden={createNetworkFinished}
+          icon={faNetworkWired}
+          onclick={createNetwork}>
+          Create
+        </Button>
+      
+        <Button
+          on:click={end}
+          class="w-full"
+          hidden={!createNetworkFinished}
+          onclick={end}>
+          Done
+        </Button>
+      </div>      
```

Comment:
```suggestion
      <div class="w-full flex flex-row">
        {#if !createNetworkFinished}
          <Button
            disabled={hasInvalidFields || createNetworkInProgress || createNetworkFinished}
            inProgress={createNetworkInProgress}
            class="w-full"
            icon={faNetworkWired}
            onclick={createNetwork}>
            Create
          </Button>
        {:else}
          <Button
            class="w-full"
            onclick={end}>
            Done
          </Button>
        {/if}
      </div>
```

some issues with the buttons, the parent component uses `space-x-4` but since one of the button is always hidden, there is some space left either on the right or the left

<img width="194" height="222" alt="Image" src="https://github.com/user-attachments/assets/c4f42257-0f53-49d6-bfc2-41576d2ac72b" />

to avoid this, we should remove the `space-x-4` and directly use Svelte if else block to either display create or done button